### PR TITLE
calculation added

### PR DESCRIPTION
**Pull request by qhdtlr**
test

### CODE REVIEW

File: another.c [hunk 1]
```diff
@@ -1 +1,2 @@
-another file from tutorial
```
Comment (qaradara) [inline another.c:1]: it is removed well

File: another.c [hunk 2]
```diff
@@ -1 +1,2 @@
...
\ No newline at end of file
+another file from tutorial
+10 + 3 = 13 is mended.
```
Comment (qaradara) [inline another.c]: 10 + 2 = 12 is required

**Review by qaradara — CHANGES_REQUESTED** [review]
Read comments please.

File: another.c (hunk 2 above)
Comment (qhdtlr) [reply another.c]: Ok.

**Review by qaradara — APPROVED** [review]
It's done well.

**Pull request merged**